### PR TITLE
new service for Ceitec Proteomics IS

### PR DESCRIPTION
**Pull request by Laliska**

### CODE REVIEW

File: gen/ceitec_proteomics [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+#forward declaration
+sub processUsers;
+sub processGroups;
+sub processMemberships;
+
+#Constants
+our $A_USER_ID;                   *A_USER_ID =               \'urn:perun:user:attribute-def:core:id';
+our $A_USER_STATUS;               *A_USER_STATUS =           \'urn:perun:member:attribute-def:core:status';
+our $A_USER_NAME;                 *A_USER_NAME =             \'urn:perun:user:attribute-def:core:firstName';
+our $A_USER_SURNAME;              *A_USER_SURNAME =			 \'urn:perun:user:attribute-def:core:lastName';
```
Comment (stavamichal) [inline gen/ceitec_proteomics]: please fix indentation

**Comment by stavamichal** [comment]
Maybe missing send script.

**Comment by Laliska** [comment]
I have added send script and repaired indentation.

**Comment by stavamichal** [comment]
Sorry, just miss click on Closing request.

**Pull request merged**